Makefile

```diff
@@ -17,10 +32,43 @@ COMMON_ARGS += --frontend-opt build-arg:GOLANG_VERSION=$(GOLANG_VERSION)
 COMMON_ARGS += --frontend-opt build-arg:SHA=$(SHA)
 COMMON_ARGS += --frontend-opt build-arg:TAG=$(TAG)
 
-all: kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+all: buildkitd builddeps kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
```

Comment:
nit: Can we collapse `buildkitd builddeps` down to `ci? It looks like `ci` has those dependencies.